times_reader/transforms.py

```diff
@@ -951,22 +1014,38 @@ def generate_dummy_processes(
             ["IMPDEMZ", "IMPDEMZ", "", "", ""],
         ]
 
-        df = pd.DataFrame(
+        ndummies = len(dummy_processes)
+
+        df1 = pd.DataFrame(
             dummy_processes,
             columns=["TechName", "TechDesc", "Tact", "Tcap", "PrimaryCG"],
         )
-        df.insert(0, "Sets", ["IMP"] * len(dummy_processes))
 
-        result = datatypes.EmbeddedXlTable(
+        df1.insert(0, "Sets", ["IMP"] * ndummies)
+
+        dummy_process_declaration = datatypes.EmbeddedXlTable(
             tag="~FI_PROCESS",
             uc_sets={},
             sheetname="",
             range="",
             filename="",
-            dataframe=df,
+            dataframe=df1,
+        )
+
+        df2 = df1[["TechName", "TechDesc"]].copy()
```

Comment:
Is it possible to give them more descriptive names?

Reply:
For df1 and df2? I tried to give descriptive names to the EmbeddedXlTables instead of the dfs.
Should I rename the dfs something like df_dummy_process_declaration and df_dummy_process_data_specification?

Reply:
Yes, df1 and df2. process_declarations and process_data_specs?

Reply:
ok, changed